scripts/update_bazel_workspace.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+# This command should be run from the emsdk root.
+
+function cleanup {
+  popd > /dev/null
```

Comment:
I don't think this is needed since that current working directory of the script doesn't effect the calling process.